src/components/ui/button.tsx

```diff
@@ -10,13 +10,13 @@ const buttonVariants = cva(
     variants: {
       variant: {
         default:
-          'bg-primary text-white hover:bg-primary-light dark:bg-slate-50 dark:text-slate-900 dark:hover:bg-slate-50/90',
+          'bg-primary text-white hover:bg-primary-light dark:bg-slate-50 dark:text-slate-900 dark:hover:bg-slate-50/90 focus-visible:ring-primary-light focus-visible:ring-offset-2',
         destructive:
           'bg-red-500 text-slate-50 hover:bg-red-500/90 dark:bg-red-900 dark:text-slate-50 dark:hover:bg-red-900/90',
         outline:
-          'border border-slate-200 bg-white hover:bg-slate-100 hover:text-slate-900 dark:border-slate-800 dark:bg-slate-950 dark:hover:bg-slate-800 dark:hover:text-slate-50',
+          'border border-primary-dark bg-white hover:bg-primary-active hover:text-primary-dark dark:border-slate-800 dark:bg-slate-950 dark:hover:bg-slate-800 dark:hover:text-slate-50 focus-visible:ring-primary-light focus-visible:ring-offset-2',
```

Comment:
🟡 Just an extra thought. I noticed that you are using variant='secondary' for the "close" button on your PlaceOrder. If you do make that switch, feel free to remove the variant='close' definition from this file since it won't be needed anymore.